style.css

```diff
@@ -783,36 +797,133 @@ textarea:focus {
   }
 }
 
-button {
-  background-color: #ff6b00;
+footer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #fff;
+  border-top: 1px solid #212121;
+}
+
+.popup_main,
+.popup_div {
+  background-color: #fff;
+  margin-left: 3.2%;
+  margin-right: 3.2%;
   padding: 3.2%;
+  display: none;
+}
+
+.popup_main {
+  z-index: 10;
+  position: fixed;
+  backdrop-filter: blur(10px);
+  top: 3.2%;
+  left: 3.2%;
+  right: 3.2%;
+  bottom: 3.2%;
+  overflow: auto;
+}
+
+.popup_divTitle {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.popup_main h2 {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.popup_topDiv ul {
+  display: flex;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
```

Comment:
For the popup modal you created, there are some issues in the design that need to be fixed to make your design match the given design:

- [ ] Please decrease the margin under the project title.
- [ ] Kindly make the project image size larger.
- [ ] The background of the popup card suppose to be blurry, kindly fix that using the `backdrop-filter` CSS property.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/ivangonzalez224/portfolio.io/assets/68796232/ea8c5caf-8673-4e51-9e4b-783eb476cae7) | ![image](https://github.com/ivangonzalez224/portfolio.io/assets/68796232/c0cba099-7592-47cb-8683-177c48c176e3)  |